draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -1701,18 +1701,19 @@ Example-Field: Baz
    through use of <xref target="RFC2047"/> encoding.
    Specifications for newly defined fields &SHOULD; limit their values to
    visible US-ASCII octets (VCHAR), SP, and HTAB.
-   A recipient &SHOULD; treat other octets in field content (obs-text) as
-   opaque data.
+   A recipient &SHOULD; treat other allowed octets in field content
+   (i.e., <x:ref>obs-text</x:ref>) as opaque data.
 </t>
 <t>
-   Field values containing CR or NUL characters are invalid and dangerous,
+   Field values containing CR, LF, or NUL characters are invalid and dangerous,
```

Comment:
I reviewed the text on field *names* and noticed that there is nothing about COLON there.  Is it worth adding an explicit prohibition there of a similar shape to this?

Reply:
Maybe, but I think that would be a different issue. It would be nice if we had one place for all these special handling requirements that are really protocol errors but have to be handled the same way by all versions.

Reply:
see #30

Reply:
Note that HTTP already limits the field-name as described, except for the lowercasing and allowing the pseudo-header-specific colon that are version-specific, and also doesn't allow token delimiters. I've added an issue on that for http2bis.